src/redist/targets/GenerateLayout.targets

```diff
@@ -106,6 +112,7 @@
     <ItemGroup>
       <BundledLayoutComponent Include="CombinedSharedHostAndFrameworkArchive">
         <BaseUrl>$(CoreSetupRootUrl)$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</BaseUrl>
+        <PrivateBaseUrl>$(PrivateCoreSetupRootUrl)$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</PrivateBaseUrl>
```

Comment:
It seems like a refactor that splits out `CoreSetupRootUrl` and `PrivateBaseUrl` would be better here, rather than adding new metadata for every file.